lib/Dialect/TTNN/Pipelines/TTNNPipelines.cpp

```diff
@@ -50,6 +50,7 @@ void createTTNNPipelineLoweringPasses(
   pm.addPass(createConvertTTIRToTTNNPass());
   // Add pass to remove unused values.
   pm.addPass(mlir::createRemoveDeadValuesPass());
+  pm.addPass(createTTNNDeallocate());
```

Comment:
nit:
Can you please add a comment above this line? If it stays like this it is like these two passes(createRemoveDeadValuesPass and createTTNNDeallocate) are attached to the comment above:
// Add pass to remove unused values.

Or insert one empty line...